src/content/en/showcase/case-study/net-a-porter.markdown

```diff
@@ -0,0 +1,115 @@
+need to surface structured data about the product to web crawlers.
+
+## Tests and success with Polymer
+
+After evaluating a number of solutions, NET-A-PORTER chose to test Google’s
+Polymer library. An initial experiment with Polymer was completed so quickly 
+and successfully that the NET-A-PORTER engineers decided to move forward 
+and introduce Polymer into the organization’s overall website development 
+planning almost immediately.
+
+Robin Glen and Matthew Green, engineers on the team, shared,
+
+<blockquote> "Polymer was easy to integrate into our system and get up and running in no time at 
+ all. The fact that we have such a large website with so many different aspects 
+ means it’s easy for us to have inconsistencies. Polymer allows us to create 
+ complete and complex web applications that are scalable and maintainable."</blockquote>
+
+Code standardization was easy as well, because Polymer is based on
+web components browser standards. Glen continues, 
+
+<blockquote>"Building using web 
+standards keeps our code close to the platform. This makes it easier to 
+learn how to use and build new components, ensuring performance will 
+continually get better as browsers improve, and letting us plan for the
+long-term. Because it’s based on the W3C standard, we are confident in 
+building on the platform. Soon even the need for polyfills will broadly go away."</blockquote>
+
+And there were SEO benefits, too. Glen says that the 
+
+<blockquote>"components
+effectively surface detailed structured data about products to search 
+engines. Testing this structured data is now directly part of our team's 
+continuous integration scheme."</blockquote>
+
+By making the up-front investment in designing and building components,
+the NET-A-PORTER team is seeing dramatic long-term efficiency gains: 
+New pages and features are simpler and faster to create and launch, and 
+it’s easier to integrate stakeholder feedback and design tweaks into the 
+engineering process.
+
+## The future of Polymer
+
+The team is thrilled with how Polymer has enabled its web development.
+Glen says, 
+
+<blockquote>"As soon as we saw how we could reuse components by 
+putting them together in different ways, we got excited. We started to 
+wonder if there were further ways in which we could exploit components 
+within the business."
+</blockquote>
+
+Glen feels there is much more to explore with Polymer. He says, 
+
+<blockquote>"This
+is just the beginning of our Polymer journey. We are currently improving 
+the delivery and maintainability of existing components, while working 
+towards expanding the use of Polymer to more NET-A-PORTER pages. We 
+are working towards documenting our approach with a publicly available 
+NET-A-PORTER component style guide. Polymer has also been very useful 
+for internal tooling. We recently built a set of graphing components and 
+re-platformed our entire monitoring service to use Polymer."
+</blockquote>
+
+## Results
```

Comment:
This section: (1) add periods to the bullets. (2) Maybe make these a tl:dr; at the top.